.github/workflows/build_rocks.yaml

```diff
@@ -88,6 +89,47 @@ jobs:
       matrix:
         path: ${{ fromJSON(needs.get-rocks.outputs.rock-paths) }}
     steps:
+      - uses: actions/checkout@v4
+        if: inputs.rockcraft-repository != ''
+        with:
+          repository: ${{ inputs.rockcraft-repository }}
+          ref: ${{ inputs.rockcraft-ref }}
+          path: rockcraft
+          fetch-depth: 0
+      - name: Get Rockcraft SHA
+        if: inputs.rockcraft-repository != ''
```

Comment:
Should the inputs.... be within ${{ }}?

Reply:
Same with other inputs :D

Reply:
The `${{ }}` can be omitted in the `if` statement: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif